src/Web/Slack/Types/File.hs

```diff
@@ -65,6 +67,7 @@ instance FromJSON Preview where
 
 data Preview = Preview { _previewText :: Text, _previewHighlight :: Text, _lines :: Int, _linesMore :: Int } deriving Show
 data FileUrl = URL { _private :: Text, _privateDownload :: Text } deriving Show
+data FileReference = FileReference { _id :: FileId } deriving Show
```

Comment:
Please call this `_fileReferenceId` and then generate lenses as for the other datatypes (see examples on the lines below).